models/review.go

```diff
@@ -416,7 +483,7 @@ func GetReviewerByIssueIDAndUserID(issueID, userID int64) (review *Review, err e
 func getReviewerByIssueIDAndUserID(e Engine, issueID, userID int64) (review *Review, err error) {
 	review = new(Review)
 
-	if _, err := e.SQL("SELECT * FROM review WHERE id IN (SELECT max(id) as id FROM review WHERE issue_id = ? AND reviewer_id = ? AND type in (?, ?, ?))",
+	if _, err := e.SQL("SELECT * FROM review WHERE id IN (SELECT max(id) as id FROM review WHERE issue_id = ? AND reviewer_id = ? AND original_author_id = 0 AND type in (?, ?, ?))",
```

Comment:
What's the reason behind the original_author_id = 0 lock here?

Reply:
Because if an original author who are not in your system , will use the id of user who do migration as ``reviewer_id``. It will break permission check later, I will add other function to get the original author's reviews, then the issue #11705 will  be fully fixed. :)

Reply:
That makes sense. Thanks. 

Add more comments to explain things - it makes things much easier to understand.